### ISSUE TITLE
It does not use `ignore` setting of .template-lintrc

### ISSUE THREAD

**Issue by mydea** [lifeart/ember-language-server#156]
We have configured template-lint to skip the tests directory (see https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/ignore.md). However, the VSCode extension still marks the hbs templates in my tests as errors. It would be nice if the extension would use the `ignore` setting from my .template-lintrc file.

**Comment by lifeart** [comment]
@mydea correct, `ts/js` files linting is not implemented in `template-lint`, and workaround located in language server.
Linking issue: https://github.com/ember-template-lint/ember-template-lint/pull/888

**Comment by lifeart** [comment]
this template-lint config should work

```
overrides: [
    {
      files: ['**/*-test.js'],
      rules: {
        'no-inline-styles': false,
      },
    },
  ],
```

**Comment by lifeart** [comment]
one more related issue https://github.com/lifeart/vscode-ember/issues/22

likely we should add configuration example into `readme.md`
